content/millennium/r4/individuals/related-person.md

```diff
@@ -109,13 +125,25 @@ List an individual RelatedPerson by its id:
 <%= headers status: 200 %>
 <%= json(:r4_relatedperson_entry) %>
 
+### Example - Encounter level
```

Comment:
Should we capitalize 'level'?

Reply:
We might wanna keep this consistent with what we do in Coverage. i.e. 'Patient-level' and 'Encounter-level'

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/526/commits/b3d1178567f0c571011e9524525c7a7d6e69fd6a#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR78